content/release-2024-0/announcements-2024-0.md

```diff
@@ -20,6 +20,121 @@ This change affects the following APIs:
+
+##### How to enable it
+Use the tenant options to create a category named `measurement.series.latestvalue` with a PUT request to a [tenant options category](https://cumulocity.com/api/core/#operation/putCategoryOptionResource).
+Example: 
+```
+PUT /tenant/options/measurement.series.latestvalue
+{
+  "c8y_Humidity.H":"", // to enable single series c8y_Humidity.H
+  "c8y_Temperature.*":"", // to able series under fragment c8y_Temperature
+  // or "*":"" to enable all 
+}
+```
+where the key is a filter of measurement series that must be persistent and its value must always be an empty string (left for a future use case).
+
+##### How it works
+
+If a measurement is created with a series that matches the configuration the device managed object
+is updated with the last series sent to the platform.
+Example:
+
+If you send 
+```
+POST /measurement/measurements
+{
+  "source":"5413"
+  "time":"2024-02-01T10:00:00Z"
+  "c8y_Temperature":{
+     "T": {
+        "value": 15,
+        "unit":"C"
+     }
+  }
+  "c8y_Speed":{
+    "S": {
+      "value": 15,
+      "unit":"m/s"
+    }
+  }
+}
+```
+then,  considering the example configuration, only `c8y_Temperature.T` is stored as part of the device, while `c8y_Speed.S` is ignored.
+This means, that the measurement is stored like before, only the state update is skipped.
+To read the latest values on device level you must use the Inventory API.
+To get a single device:
+```
+GET /inventory/managedObjects/5413?withLatestValues=true
+{
+   ...
+   "c8y_LatestMeasurements":{
+        "c8y_Temperature":{
+           "T":{
+             "value":15,
+             "time":"2024-02-01T10:00:00Z",
+             "unit":"C"
+           }
+        }
+   }
+}
+
+```
+To get a list of devices matching the expected criteria, 
+for example, get all devices which have a reported temperature higher than 10 degrees:
+
+```
+GET /inventory/managedObjects?withLatestValues=true&query=$filter=c8y_LatestMeasurements.c8y_Temperature.T.value+gt+10
+{
+  managedObjects: [
+    {
+        ...
+        "c8y_LatestMeasurements":{
+            "c8y_Temperature":{
+                "T":{
+                    "value":15,
+                    "time":"2024-02-01T10:00:00Z",
+                    "unit":"C"
+                }
+            }
+        }
+    }
+  ]
+}
+
+```
+##### Implications
+
+This feature introduces an additional operation upon measurement creation.
+This results in performance degradation, depending on the number of series to be
+stored in each measurement, reaching from 5% for single series in each measurement to 
+more than 20% in case of 50 series per measurement.
+
+#### Limitations
+
+##### Security
+
+The latest measurement values are part of the managed object and they follow the managed object inventory role permissions.
+It's not respecting the inventory roles for measurements. 
```

Comment:
```suggestion
The latest measurement values are part of the managed object and they follow the managed object inventory role permissions instead of respecting the inventory roles for measurements. 
```